packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -373,6 +390,15 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
         }
     }
 
+    /**
+     * force refresh workspace in navigator
+     */
+    async refreshWorkspace(): Promise<void> {
+        const { model } = await this.widget;
+        const root = model.root as CompositeTreeNode;
```

Comment:
Should not we refresh the whole model?

i.e. `await model.refresh();`

Reply:
Thanks, it's my mistake, `model.refresh()` can refresh the root node